src/librbd/image/CloneRequest.cc

```diff
@@ -99,7 +98,6 @@ void CloneRequest<I>::validate_options() {
       complete(-ENOSYS);
       return;
     }
-    m_use_p_features = false;
```

Comment:
Huh? If we explicitly set the new features to use, we shouldn't inherit features from the parent.